lib/src/genius.dart

```diff
@@ -594,4 +593,42 @@ class Genius {
     }
     return artistFound;
   }
+
+  /// return the most 10 popular's `Song` where given artist is a primary artist
+  ///
+  ///  `artistName`  primary artist name
+  ///
+  /// `includeLyrics` if [true] , it will call [lyrics] to get songs lyrics, make it true only when necessary , because the function slower
+  ///
+  ///
+  Future<List<Song>> searchArtistMostPopularSongs({
```

Comment:
a `bool getFullInfo` (that defaults to false, maybe) parameter should be added so it is consistent with the functions that search for songs as well.

Reply:
i dont think it's a good idea , but this bool is not about full info , some fields could be null , so i think this name make sense, its about lyric .

Reply:
Sorry, I was not clear! I am not talking about the name of the `includeLyrics` parameter (this is fined), I am requesting a new parameter (`bool getFullInfo` just like in the `searchSong` function) to be added and the make change changes in function accordingly.